### PR TITLE
feat(helm): add RollingUpdate parameters

### PR DESCRIPTION
**Pull request by gforien**
### SUMMARY
We want to handle the RollingUpdate parameters for the supersetNode and supersetWorker.

### How
- Add the parameters in the `deployment.yaml` and `deployment-worker.yaml` templates
- Get default values from [K8S documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#deploymentstrategy-v1-apps) to ensure no change
- Put default values in `values.yaml`
- Update JSON schema
- Bump chart version

### TESTING INSTRUCTIONS
```sh
helm lint
helm template . superset-release
```

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [x] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: helm/superset/Chart.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.6.6
+version: 0.6.7
```
Comment (craig-rueda) [inline helm/superset/Chart.yaml:25]: This should be `0.7.1`

File: helm/superset/templates/deployment-worker.yaml [hunk 2]
```diff
@@ -33,6 +33,11 @@ spec:
     matchLabels:
       app: {{ template "superset.name" . }}-worker
       release: {{ .Release.Name }}
+  strategy:
+    type: RollingUpdate
```
Comment (craig-rueda) [inline helm/superset/templates/deployment-worker.yaml:37]: Pls make the strategy configurable. You can just add the whole `strategy` section into `values.yaml` and then call `toYaml` here.

**Comment by dpgaspar** [comment]
Hi @gforien would be great if you could rebase and address the requested changes. Thank you!

**Comment by gforien** [comment]
Hi @dpgaspar
Sorry for the delay. I will do this in the upcoming days

**Comment by gforien** [comment]
Hi 👋 
I have rebased my branch, and addressed the requested changes
Thank you for your patience

**Comment by gforien** [comment]
@craig-rueda @dpgaspar

File: helm/superset/Chart.yaml [hunk 3]
```diff
@@ -22,7 +22,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.7.2
+version: 0.7.3
```
Comment (craig-rueda) [inline helm/superset/Chart.yaml:25]: Prob need to bump this one more time

**Review by craig-rueda — APPROVED** [review]
LGTM

File: helm/superset/Chart.yaml (hunk 3 above)
Comment (gforien) [reply helm/superset/Chart.yaml]: Did not know which one you would merge first :)

**Pull request merged**